File: app/src/main/java/com/magenta/rx/java/model/record/DictionaryAnswer.java
package com.magenta.rx.java.model.record;

public class DictionaryAnswer {

    private Object head;
    private Definition[] def;

    public DictionaryAnswer(String head, Definition[] def) {
        this.head = head;
        this.def = def;
    }

    public DictionaryAnswer() {
    }

    public Object getHead() {
        return head;
    }

    public void setHead(String head) {
        this.head = head;
    }

    public Definition[] getDef() {
        return def;
    }

    public void setDef(Definition[] def) {
        this.def = def;
    }
}

File: app/src/main/kotlin/com/magenta/rx/kotlin/record/DictionaryAnswer.kt
package com.magenta.rx.kotlin.record

class DictionaryAnswer(private var head: Any?, private var def: Array<Definition>) {

    fun getHead() = head

    fun getDef() = def

    fun setHead(head: Any) {
        this.head = head
    }

    fun setDef(def: Array<Definition>) {
        this.def = def
    }
}